andela/ah-frontend-thor
# #161382350 Users should be able to presented with a reset password link

#### What does this PR do?
Provide a user interface to enable the user reset his/her password

#### Description of Task to be completed?
Users should be able to presented with a reset password link

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - start the server using `yarn start:dev`
 - Enter `localhost:8080/forgot_password` in the browser
 - Enter the email and submit
 - Check the inbox of the email address provided
 - Click on the link in the inbox
 - You will be redirected to another page to provide the new password
 - Enter the new password and submit(the link can only be used once)

#### Any background context you want to provide?
The link in the backend should redirect the user to the user interface in the frontend

#### What are the relevant pivotal tracker stories?
#161382350

#### Screenshots (if appropriate)

Page to allow user provide email to which the reset link is sent
<img width="1440" alt="nefg" src="https://user-images.githubusercontent.com/9901086/49799408-e9b80980-fd55-11e8-9386-ab6dccf5aa5b.png">



Page to allow user put their new password
<img width="1440" alt="newcp" src="https://user-images.githubusercontent.com/9901086/49799329-bbd2c500-fd55-11e8-8b4f-485c82c80b7f.png">
